src/CommandingOfficers/Commander.java

```diff
@@ -300,17 +300,17 @@ public int calculateCombatCharge(UnitDelta minion, UnitDelta enemy, boolean isCo
     if( minion == null || enemy == null )
       return 0;
 
-    int myHPLoss  = minion.getHPDamage();
-    int myHPDealt = enemy.getHPDamage();
+    int guiHPLoss  = minion.getHPDamage() / 10;
+    int guiHPDealt =  enemy.getHPDamage() / 10;
```

Comment:
Shouldn't `getHPDamage()` already be returning the (0-10) number, thus no need to divide by 10?

Reply:
Nope, that's the base damage from the damage chart.
The only thing in (0,10] now is `getHPFactor()` and whatever is explicitly converted.

Reply:
Oh. Well that's just confusing naming all around.